AtlasLootClassic_Collections/data.lua

```diff
@@ -1495,14 +1495,17 @@ data["Brewfest"] = {
 				{ 10,  33047 }, -- Belbi's Eyesight Enhancing Romance Goggles (Alliance)
 				{ 11,  34008 }, -- Blix's Eyesight Enhancing Romance Goggles (Horde)
 				{ 13,  33016 }, -- Blue Brewfest Stein
-				{ 16,  32233 }, -- Wolpertinger's Tankard
-				{ 18,  37599 }, -- "Brew of the Month" Club Membership Form
-				{ 20,  37829 }, -- Brewfest Prize Token
-				{ 22,  33927 }, -- Brewfest Pony Keg
-				{ 24,  37750 }, -- Fresh Brewfest Hops
-				{ 25,  39477 }, -- Fresh Dwarven Brewfest Hops
-				{ 26,  39476 }, -- Fresh Goblin Brewfest Hops
-				{ 27,  37816 }, -- Preserved Brewfest Hops
+				{ 15,  37829 }, -- Brewfest Prize Token
+				{ 16,  33976 }, -- Brewfest Ram
+				{ 17,  33977 }, -- Swift Brewfest Ram
+				{ 19,  32233 }, -- Wolpertinger's Tankard
+				{ 21,  34028 }, -- "Honorary Brewer" Hand Stamp
```

Comment:
Are you sure this is really in the Game?
According to [wowhead comments](https://tbc.wowhead.com/item=33978/honorary-brewer-hand-stamp#comments), you would buy the Hand Stamp with 600 Tokens from _Blix Fixwidget_ (H) / _Belbi Quikswitch_ (A). Then with the Quest done, you would be able to buy `Brewfest Ram` or `Switft Brewfest Ram`.
The comments say that this was only doable in the first Brewfest and in the following Brewfests you would loot the `Swift Brewfest Ram` from Coren Direbrew, with `Brewfest Ram` beeing no longer available.

Im in the game right now and the vendor is not selling the `"Honorary Brewer" Hand Stamp` + the `Swift Brewfest Ram` drops from Coren Direbrew

Reply:
I play as a horde and Blix Fixwidget sells ["Honorary Brewer" Hand Stamp] for 600 tokens, hadnt done the quest cause i don't really care about mounts but probably boths rums are obtainable from the vendor as the quest item is live now. There is even a wowhead article about it.
The think is that i don't know if Coren also drops the Ram, cant find any info about it.

Reply:
![image](https://user-images.githubusercontent.com/85525377/134183757-c3f2f62e-6a98-4916-b4a2-c994ed899c2f.png)

Reply:
Oh wow 😮
I looked into it yesterday bcs i wanted to know if i could get this, but it wasnt there.
Seems like they brought it in over night?🤩